src/ui/src/partials/admin_system.html

```diff
@@ -23,8 +23,79 @@ <h1 class='page-header'>
   {{alert.msg}}
 </div>
 
-<div id="systems-container"
-     class="animate-if"
+<div ng-show="runners.length>0">
+  <h3>
+    <span>Runners</span>
+  </h3>
+
+  <div class="panel-container animate-if"
+      ng-if="responseState(response) === 'success'">
+
+    <div ng-repeat="runners in groupedRunners"
+        class="container-item panel panel-default"
+        style="display: block;">
+
+      <div class="panel-heading" style="font-size: 22px;">
+        <span>{{runners[0].path}}</span>
+
+        <div class="btn-toolbar pull-right">
+          <div class="btn-group btn-group-sm">
+            <button type="button" class="btn btn-default fa fa-play"
+                    ng-show="hasPermission(user, 'bg-system-admin')"
+                    ng-click="startSystem(system)"
```

Comment:
If this works I will be surprised. I don't see the ng model for system here.

Reply:
It won't. This is why it's in draft 😄